wayland/bunsenlabs-session-wayland.1

```diff
@@ -0,0 +1,66 @@
+.TH "BUNSENLABS-SESSION-WAYLAND" "1"
+.SH "NAME"
+bunsenlabs-session-wayland \(em Starts a BunsenLabs Wayland session.
+
+.SH "DESCRIPTION"
+.PP
+\fBbunsenlabs-session-wayland\fR runs a BunsenLabs Wayland session,
+calling a compositor and starting the necessary auxiliary applications.
+There is no full session manager, so you will not be able
+to save your state from one log in to the next.
+Although labwc is the default BunsenLabs compositor or window manager,
+users can select others while continuing to use the BunsenLabs
+menu, keyboard shortcuts and autostarted applications (see below).
+.PP
+\fBbunsenlabs-session-wayland\fR does not take any command line
+arguments.
+.PP
+\fBbunsenlabs-session-wayland\fR is set as the Debian alternative
+"wayland-session-manager" so it will be run by default in a Debian wayland
+session, and a .desktop file is installed to /usr/share/wayland-sessions
+so it can be started by a display manager like LightDM.
+.PP
+On log in, \fBbunsenlabs-session-wayland\fR will run the
+~/.config/bunsen/autostart-wayland script if it exists, and will copy in the
+default file from /usr/share/bunsen/skel/.config/bunsen/autostart otherwise. You may
```

Comment:
default file is /usr/share/bunsen/skel/.config/bunsen/ autostart-wayland

Reply:
missed that one, fix in next commit